bagtrekkin/tests/test_searchForm.py

```diff
@@ -0,0 +1,94 @@
+from django.test import TestCase
+from django.test import Client
+from django.core.urlresolvers import reverse
+from django.test.utils import setup_test_environment
+
+from bagtrekkin.forms import SearchForm
+from bagtrekkin.models import Passenger, Luggage, Log
+
+
+class SearchFormTestCase(TestCase):
+    fixtures = ['airports', 'companies', 'passengers', 'flights', 'users', 'employees', 'luggages', 'logs']
+
+    def setUp(self):
+        super(SearchFormTestCase, self).setUp()
+        self._passenger = Passenger.objects.get(pk=1)
+        self._luggages = Luggage.objects.filter(passenger=self._passenger)
+        self._logs = Log.objects.filter(luggage__passenger=self._passenger)
+        self.client = Client()
+        self.client.login(username='capflam', password='123')
+
+    def test_init_no_data(self):
+        """Init should be successful without provided data"""
+        form = SearchForm()
+        self.assertTrue(isinstance(form, SearchForm))
+
+    def test_init_pnr_data(self):
+        """Init should be successful with pnr provided data"""
+        form = SearchForm({'pnr': 'YSVI82'})
```

Comment:
Possible to factors both inits in one method. 
See https://github.com/goujonpa/bagtrekkin/blob/roddehugo_tastypie_tests/bagtrekkin/api/tests.py#L32-L36
Create a method with a name not starting with test which takes to arguments `key` and `value`
Then both init tests will call that method ;)